### PR TITLE
Refactor operator modal + misc refactoring

### PR DESCRIPTION
**Pull request by mondoreale**
`BecomeOperatorModal` is now simply `OperatorModal`. On top of that, it comes with a single toaster instance, waits for blocks and takes the user to the Operator page after they create one.

This refactoring is part of the "wait for block as long as it takes" shift.

### CODE REVIEW

File: src/hooks/operators.ts [hunk 1]
```diff
@@ -551,3 +534,60 @@ const mapOperatorOrder = (orderBy: string | undefined): Operator_OrderBy => {
             return Operator_OrderBy.Id
     }
 }
+
+/**
+ * Returns a callback that takes the user through the process of creating
+ * an operator or updating an existing operator.
+ */
+export function useTouchOperatorCallback() {
```
Comment (tumppi) [inline src/hooks/operators.ts:542]: I'm not sure about the name of this callback 🤔 I'd say `touching` makes me think of updating only timestamps or so.

**Review by tumppi — APPROVED** [review]
It's clearer this way 👍

File: src/hooks/operators.ts (hunk 1 above)
Comment (mondoreale) [reply src/hooks/operators.ts:542]: Yeah, you may be right, I need a better name.

Comment (mondoreale) [reply src/hooks/operators.ts:542]: `useSaveOperator`, `usePersistOperator`, any other ideas?

Comment (tumppi) [reply src/hooks/operators.ts:542]: I'd vote for good old `useSaveOperator` 😄

**Comment by mondoreale** [comment]
Side-tracked it a bit into a larger refactoring. More like a chore with plenty simplifications, automations and some dead code shake off.

**Review by mondoreale — COMMENTED** [review]
Also,
- `isEthereumAddress` is gone. I use `isAddress` directly, and
- `saveOperator` is a utility, not a hook-driven callback.

**Pull request merged**